src/system-plugins/platform-manager/platforms/mock/boards/mock3000/bridge.js

```diff
@@ -357,29 +360,48 @@ function Bridge()
     //p(t) = 90*sin(t)
     currentPitch = 90*Math.sin(time);
     result += 'imu_p:' + encode( currentPitch ) + ';';
+
     // Generate roll
     currentRoll = Math.floor(Math.random()*91) - 45;
     result += 'imu_r:' + encode( currentRoll )+ ';';
+
     // Generate battery tube 1 current
     rnd = (Math.random() * 20 - 10) / 100;
     current += current * rnd;
     current = Math.min(Math.max(current, 1), 10);
     result += 'BT1I:' + current + ';';
+
     // Generate battery tube 2 current
     rnd = (Math.random() * 20 - 10) / 100;
     current += current * rnd;
     current = Math.min(Math.max(current, 1), 10);
     result += 'BT2I:' + current + ';';
+
```

Comment:
Made mock voltage ramp up and down between 5V and 12V so you can see the whole range of values exercised.